schemas/dab.draft.schema.json

```diff
@@ -357,11 +357,34 @@
               "required": [
                 "connection-string"
               ]
+            },
+            "open-telemetry": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "endpoint": {
+                  "type": "string",
+                  "description": "Open Telemetry connection string"
+                },
+                "headers": {
+                  "type": "string",
+                  "description": "Open Telemetry headers"
+                },
```

Comment:
How do I provide multiple headers? Should this be a dictionary or are they meant to be tokenized (like, comma separated or something)?

Reply:
is headers necessary? is endpoint alone not enough?

Reply:
if we don't need to parse headers and can be directly used as string, then comma separated should be fine.

Reply:
Can `headers` be name/value?

```json
{
    "headers": {
        "name1": "value1",
        "name2": "value2",
        "name3": "value3"
    }
}
```

That would be easy.  Or perhaps a little more typed, like this:

```json
{
    "headers": [
        { "name": "name1", "value": "value1" },
        { "name": "name2", "value": "value2" },
        { "name": "name3", "value": "value3" }
    ]
}
```

The second one is slightly better in case we need a third value someday.

Reply:
It would be double parsing since OTLP config requires a string as header (string-input --> structure, structure --> string output). Also, the headers for OTLP don't have a defined structure.